target/linux/sunxi/cortexa53/config-5.4

```diff
@@ -153,3 +153,10 @@ CONFIG_THREAD_INFO_IN_TASK=y
 CONFIG_UNMAP_KERNEL_AT_EL0=y
 CONFIG_VMAP_STACK=y
 CONFIG_ZONE_DMA32=y
+
+# Config for usb-otg controlller in host mode
```

Comment:
That's not how these files work. At certain points, e.g. after a kernel bump or major changes, the files are updated with `make kernel_oldconfig`, and then new symbols will be added, old ones removed, and everything will be sorted alphabetically. So, the kind of grouping you do here won't be persistent.

I'd rather recommend to run make kernel_oldconfig yourself and check whether other options are affected by your changes.

Apart from that: Do you need all of that to be added via config, or can you use modules instead?
What's the size impact of these changes?